doc/stages/filters.rst

```diff
@@ -227,6 +228,9 @@ the input. These filters will invalidate an existing KD-tree.
     Remove points that are in a raster cell but have a value far from the
     value of the raster.
 
+:ref:`filters.firstinvoxel`
+    Filter out points that aren't the first encountered that lie in a voxel.
```

Comment:
Only comment on the PR is that this wording is bit awkward still. Maybe "Retain only the first detected point per voxel."